### ISSUE TITLE
Version 1.1.3 is broken

### ISSUE THREAD

**Issue by sven1103** [qbicsoftware/openbis-client-lib#24]
Using the library and creating an `OpenBISClient` instance produces this exception, which does not occur in version `1.1.2`:

```java
Caught: java.lang.NoClassDefFoundError: ch/systemsx/cisd/common/exceptions/InvalidSessionException
java.lang.NoClassDefFoundError: ch/systemsx/cisd/common/exceptions/InvalidSessionException
	at life.qbic.Query.main(Query.groovy:11)
Caused by: java.lang.ClassNotFoundException: ch.systemsx.cisd.common.exceptions.InvalidSessionException
	... 1 more
```

**Comment by chahuistle** [comment]
Fixed in version 1.1.4 https://github.com/qbicsoftware/openbis-client-lib/releases/tag/1.1.4

**Comment by chahuistle** [comment]
the problem was that version 1.1.3 did not include any openBIS dependencies, but version 1.1.4 does.
